using-valtimo/form-flow/create-form-flow-definition.md

```diff
@@ -17,119 +17,157 @@ To create form flow definition, the following steps are necessary:
 
   Which properties are required to be set depends on the step type.
 
-    ```json
-    {
-        "startStep": "start",
-        "steps": [
-            {
-                "key": "start",
-                "type": {
-                    "name": "form",
-                    "properties":  {
-                        "definition": "start-form"
-                    }
-                },
-                "nextStep": "step2"
+```json
+{
+    "startStep": "personalDetailsStep",
+    "steps": [
+        {
+            "key": "personalDetailsStep",
+            "type": {
+                "name": "form",
+                "properties": {
+                    "definition": "personal-details-form"
+                }
             },
-            {
-                "key": "step2",
-                "type": {
-                    "name": "form",
-                    "properties":  {
-                        "definition": "step2-form"
-                    }
+            "nextSteps": [
+                {
+                    "step": "loanApprovedStep",
+                    "condition": "${#jsonPath(step.submissionData, '$.personalDetails.age') >= 21}"
                 },
-                "nextSteps": [
-                    {
-                        "step": "step3"
-                    },
-                    {
-                        "step": "end"
-                    }
-               ]
+                {
+                    "step": "loanDeniedStep"
+                }
+           ]
+        },
+        {
+            "key": "loanApprovedStep",
+            "type": {
+                "name": "form",
+                "properties":  {
+                    "definition": "loan-approved-form"
+                }
             },
-            {
-                "key": "step3",
-                "type": {
-                    "name": "form",
-                    "properties":  {
-                        "definition": "step3-form"
-                    }
-                },
-                "nextStep": "end"
+            "nextStep": "summaryStep"
+        },
+        {
+            "key": "loanDeniedStep",
+            "type": {
+                "name": "form",
+                "properties":  {
+                    "definition": "loan-denied-form"
+                }
             },
-            {
-                "key": "end",
-                "type": {
-                    "name": "form",
-                    "properties":  {
-                        "definition": "end-form"
-                    }
+            "nextStep": "summaryStep"
+        },
+        {
+            "key": "summaryStep",
+            "type": {
+                "name": "form",
+                "properties":  {
+                    "definition": "summary-form"
                 }
             }
-        ]
-    }
-    ```
+        }
+    ]
+}
+```
 
 * Add triggers to steps where necessary (e.g. to store data externally). See below for more information.
```

Comment:
Please state the names of the triggers 'onOpen', 'onBack' that the user should look for in the example. Now it's a guessing game which 'triggers' you are referring to.

Reply:
The information below states what the triggers are. Its collapsed